sass/components/_bundle.scss

```diff
@@ -0,0 +1,102 @@
+@mixin background-horiz-line($background-color: color('primary'), $line-color: border('primary')) {
+  background-image: linear-gradient($background-color 49%,
+                    $line-color 49%,
+                    $line-color 50%,
+                    $background-color 50%,
+                    $background-color 100%);
+}
+
+@mixin bundle-theme($font-color, $border-color) {
+  color: $font-color;
+  border-color: $border-color;
+  background-color: transparent;
+}
+
+%table {
+  font-family: $secondary-font;
+  font-size: $font-regular;
+  font-weight: $font-weight-l;
+  text-align: center;
+  border-collapse: collapse;
+  border-width: $input-border-width;
+  border-style: $input-border-style;
+}
+
+.bundle {
+  @extend %table;
+  min-width: 20em;
+  @include bundle-theme(text('primary'), border('primary'));
+
+  th {
+    padding: 2.2rem 4rem;
+    font-weight: $font-weight-sb;
+  }
+
+  tbody tr:first-child {
+    display: block;
+    @include background-horiz-line($line-color: border('primary'));
+  }
+
+  td {
+    padding: .667rem;
+    font-family: $primary-font;
+    font-size: $font-small;
+  }
+
+  .bundle-plan {
```

Comment:
Does `.bundle-plan` _need_ to be nested inside `.bundle`? Keep in mind the selector chain that will be generated:

```
.bundle .bundle-plan span { ... }
```

Nest only as deep as necessary, stay as shallow as possible.